src/main/resources/assets/storagedrawers/lang/it_it.json

```diff
@@ -0,0 +1,119 @@
+    "item.storagedrawers.creative_storage_upgrade.desc": "Spazio quasi infinito!",
+    "item.storagedrawers.creative_vending_upgrade": "Miglioramento da distributore in Creativa",
+    "item.storagedrawers.creative_vending_upgrade.desc": "Disponibilità di oggetti infinita",
+    "item.storagedrawers.conversion_upgrade": "Miglioramento della conversione",
+    "item.storagedrawers.conversion_upgrade.desc": "Converte automaticamente gli oggetti compatibili",
+    "item.storagedrawers.fill_level_upgrade": "Miglioramento per il RLvello di Riempimento",
+    "item.storagedrawers.fill_level_upgrade.desc": "Mostra quanto è pieno il deposito",
+    "item.storagedrawers.illumination_upgrade": "Miglioramento dell'illuminazione",
+    "item.storagedrawers.illumination_upgrade.desc": "Illumina le etichette dei Drawers al buio!",
+    "item.storagedrawers.redstone_upgrade": "Miglioramento Pietrarossa",
+    "item.storagedrawers.redstone_upgrade.desc": "Emette un segnale di pietrarossa quando completamente pieno",
+    "item.storagedrawers.min_redstone_upgrade": "Miglioramento pietrarossa minimo",
+    "item.storagedrawers.min_redstone_upgrade.desc": "Emette il segnale riferito allo slot meno pieno",
+    "item.storagedrawers.max_redstone_upgrade": "Miglioramento pietrarossa massimo",
+    "item.storagedrawers.max_redstone_upgrade.desc": "Emette il segnale riferito allo slot più pieno",
+    "item.storagedrawers.upgrade_template": "Modello per il miglioramento",
+    "item.storagedrawers.drawer_key": "Chiave dei Drawer",
+    "item.storagedrawers.drawer_key.desc": "Blocca e sblocca i Drawers\nMantieni in mano per bloccare gli oggetti appena piazzati",
+    "item.storagedrawers.shroud_key": "Chiave di Occultamento",
+    "item.storagedrawers.shroud_key.desc": "Mostra o nasconde le etichette degli oggetti",
+    "item.storagedrawers.quantify_key": "Chiave di Quantità",
+    "item.storagedrawers.quantify_key.desc": "Mostra o nasconde le etichette delle quantità immagazzinate",
+    "item.storagedrawers.personal_key": "Chiave Personale",
+    "item.storagedrawers.personal_key.desc": "Protegge l'accesso dagli altri giocatori",
+
+    "itemGroup.storagedrawers": "Storage Drawers",
+
+    "message.storagedrawers.cannot_add_upgrade": "Non è possibile aggiungere questo miglioramento in questo Drawer",
+    "message.storagedrawers.max_upgrades": "Questo Drawer non può accettare altri miglioramenti",
+
+    "tooltip.storagedrawers.drawers.capacity": "Conserva %d stacks",
+    "tooltip.storagedrawers.drawers.sealed": "Oggetti sigillati all'interno",
+    "tooltip.storagedrawers.waila.empty": "<Vuoto>",
+    "tooltip.storagedrawers.waila.drawer": "#%s: %s",
+    "tooltip.storagedrawers.waila.nolimit": "Limite di Stacks: ∞",
+    "tooltip.storagedrawers.waila.limit": "Limite di Stacks: %s (x%s)",
+    "tooltip.storagedrawers.waila.locked": "Bloccato",
+    "tooltip.storagedrawers.waila.void": "Vacuo",
```

Comment:
Isn't "Vuoto" better?

Reply:
it is!

Reply:
We need to find a word that let understand items will be destroyed when exceeding and "vuoto" don't fit well with that.

Reply:
@jaquadro can you give us some context for this?